python/lsst/ctrl/mpexec/singleQuantumExecutor.py

```diff
@@ -98,29 +98,27 @@ def execute(self, taskDef, quantum, butler):
         task = self.makeTask(taskClass, config, butler)
         self.runQuantum(task, quantum, taskDef, butler)
 
-    def setupLogging(self, taskClass, config, quantum):
+    def setupLogging(self, taskDef, quantum):
         """Configure logging system for execution of this task.
 
         Ths method can setup logging to attach task- or
         quantum-specific information to log messages. Potentially this can
-        take into accout some info from task configuration as well.
+        take into account some info from task configuration as well.
 
         Parameters
         ----------
-        taskClass : `type`
-            Sub-class of `~lsst.pipe.base.PipelineTask`.
-        config : `~lsst.pipe.base.PipelineTaskConfig`
-            Configuration object for this task
+        taskDef : `lsst.pipe.base.TaskDef`
+            The task definition.
         quantum : `~lsst.daf.butler.Quantum`
             Single Quantum instance.
         """
         # include input dataIds into MDC
         dataIds = set(ref.dataId for ref in chain.from_iterable(quantum.inputs.values()))
         if dataIds:
             if len(dataIds) == 1:
-                Log.MDC("LABEL", str(dataIds.pop()))
+                Log.MDC("LABEL", f"{taskDef.label}: {dataIds.pop()}")
             else:
-                Log.MDC("LABEL", '[' + ', '.join([str(dataId) for dataId in dataIds]) + ']')
+                Log.MDC("LABEL", f'{taskDef.label}:[' + ', '.join([str(dataId) for dataId in dataIds]) + ']')
```

Comment:
Not a big deal, but why the space after the colon above but not here?  Might make it messier to extract stuff automatically from the logs to have these inconsistent.

Reply:
It was a mistake. Fixed and removed the space in the first one.

Reply:
Although extraction of dataIds is going to need a tricky regex, especially if the dataId values include a `]`.

Reply:
This is where JSON logs can help...  Still, the `)(` in the longlog format should be good enough.

Reply:
Before my SSD died last night and wiped the code out, I was going to see if I can serialize those dataIds into Json form in the log. I'll try it again.